client/components/web-preview/content.jsx

```diff
@@ -406,21 +407,40 @@ export default class WebPreviewContent extends Component {
 					) }
 					{ 'seo' !== this.state.device && (
 						<div
+							onMouseEnter={ () => this.setState( { showIFrameOverlay: true } ) }
```

Comment:
Since this isn't gated behind any conditionals, does that mean that all implementations of web preview will now have an iframe overlay?

Reply:
9wpoYJrMlYKAKUYvVBMN74-fi?t=LSr3Run0HrsvQdqa-0

Reply:
That's true, I think I should do a check for Launchpad for all of these changes